src/DataSource.ts

```diff
@@ -73,7 +76,7 @@ export class DataSource extends DataSourceApi<MyQuery, MyDataSourceOptions> {
       payload = payload.replace(/\$timeFrom/g, range.from.valueOf().toString());
       payload = payload.replace(/\$timeTo/g, range.to.valueOf().toString());
     }
-    payload = getTemplateSrv().replace(payload, scopedVars);
+    payload = this.templateSrv.replace(payload, scopedVars);
```

Comment:
`getTemplateSrv()` is the new, documented way to access the current `templateSrv` for variable interpolation... https://grafana.com/docs/grafana/latest/developers/plugins/add-support-for-variables/#interpolate-variables-in-data-source-plugins

Reply:
so to complete that thought, if you encountered an error trying to import that from `@grafana/runtime` you probably don't have the latest version installed.